ntbs-service/Pages/Notifications/OverviewPartials/_TreatmentEventsOverviewPartial.cshtml

```diff
@@ -48,14 +48,20 @@
             var treatmentEvents = Model.GroupedTreatmentEvents[episodeNumber];
             
             <div>
-                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4><span>@treatmentEvents.First().FormattedEventDate - @treatmentEvents.Last().FormattedEventDate </span>
+                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4>
+                    <span>@treatmentEvents.First().FormattedEventDate
+                        @if (episodeNumber != Model.GroupedTreatmentEvents.Count)
+                        {
+                            <span> - @treatmentEvents.Last().FormattedEventDate</span>
+                        }
+                    </span>
```

Comment:
I think the nested span is a little clunky, and would probably just construct the string before, or unroll the nesting and do if () { <span> simple </span> } else { <span> slightly longer </span> }